src/md/docs.md

```diff
@@ -1,121 +1,43 @@
 # Documentation
 
-- [Primary documentation](/docs/apex/) for Apache Apex including overviews of the product,
-security, application development, operators and the commandline tool.
+- **[Apache Apex Core Documentation](/docs/apex/)** - overview of Apache Apex, security, application development, operators and the commandline tool.
+- **[Apache Apex Malhar Documentation](/docs/malhar/)** - operator library overview and in-depth tutorials for select operators, (such as Kafka Input).
+- **<a href="https://www.datatorrent.com/docs/apidocs/" rel="nofollow">Java API documentation</a>** - covering both the Apex Core and Apex Malhar library.
 
```

Comment:
Why remove text ? Suggest avoiding gratuitous rewrites unless there is a good reason.